### PR TITLE
A slight conversation change

### PR DESCRIPTION
**Pull request by Turtleroku**
A slight change to the conversation with Ijs about the mix of isotopes. I think it sounds a little better with ratio as isotopes and elements are redundant.

### CODE REVIEW

File: data/free worlds intro.txt [hunk 1]
```diff
@@ -751,8 +751,8 @@ mission "FW Katya 2B"
 	clearance
 	passengers 1
 	npc
+		personality nemesis plunder waiting
```
Comment (tehhowch) [inline data/free worlds intro.txt]: ?

File: data/free worlds intro.txt [hunk 2]
```diff
@@ -1575,7 +1575,7 @@ mission "FW Escort Second Chance"
 	description "Return to <destination> to meet up with Tomek Voigt and see if he will forgive you for failing the escort mission he gave you."
 	minor
 	source
-		near Sabik 1 100
+		near Sabik 1 1000
```
Comment (tehhowch) [inline data/free worlds intro.txt]: ?

File: data/free worlds intro.txt (hunk 1 above)
Comment (Turtleroku) [reply data/free worlds intro.txt]: yea I'm not sure why those showed up. I just copied the newest version of the file before I submitted the pull request.

**Comment by Turtleroku** [comment]
I am not sure why those showed up as I copied the file from Endless-sky master branch about a min before I submitted the pull. I fixed it to agree with master.

**Comment by tehhowch** [comment]
Your original endless sky master branch was likely not up to date, or you copy-and-pasted the whole file content and did not review the changes prior to committing.

**Comment by Turtleroku** [comment]
tehhowch, what is easiest way to update my forked master branch?

**Comment by Bladewood** [comment]
Create a new branch on your fork. Edit and make pull requests with that. Delete the branch after a decision on the pull request is made (merge or reject).

**Comment by endless-sky** [comment]
This change doesn't seem to be an improvement, to me. It's just a matter of opinion which one is better.

**Pull request closed without merging**